### PR TITLE
Add element.py file

### PR DESCRIPTION
**Pull request by rodrigomoliveira1**
This file should replace the main `class Element` from the shaft_element.py, disk_element.py, bearing_seal_element.py files

### CODE REVIEW

File: ross/bearing_seal_element.py [hunk 1]
```diff
@@ -5,33 +5,13 @@
 import matplotlib.pyplot as plt
 import matplotlib.patches as mpatches
 from abc import ABC
+from ross.ross.element.py import *
```
Comment (raphaeltimbo) [inline ross/bearing_seal_element.py]: I don't think this import will work. 
Try:
`from element import Element`
In this case is a good practice to be explicit on the imports and not use `import *`

File: ross/element.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+from abc import ABC
```
Comment (raphaeltimbo) [inline ross/element.py]: The `abstractmethod` should be imported from abc:
```python
from abc import ABC, abstractmethod
```

File: ross/bearing_seal_element.py (hunk 1 above)
Comment (rodrigomoliveira1) [reply ross/bearing_seal_element.py]: Done!

- [shaft_element.py](https://github.com/ross-rotordynamics/ross/commit/14e8f8b2b701d02012c5228baabd522d6897253f)

- [disk_element.py](https://github.com/ross-rotordynamics/ross/commit/690e8ae9f7034da69b4dec034113f323fce429b9)

- [beraing_seal_element.py](https://github.com/ross-rotordynamics/ross/commit/65c71fba65aecd9772d53627faa5bbe5ea807ef5)

File: ross/element.py (hunk 2 above)
Comment (rodrigomoliveira1) [reply ross/element.py]: Done.

- [element.py](https://github.com/ross-rotordynamics/ross/commit/9bc56eeef8c788cc605ab47ae84fe549eed804e1)

**Comment by raphaeltimbo** [comment]
Closes #19 .

**Pull request merged**